slog.go

```diff
@@ -0,0 +1,70 @@
+package watermill
+
+import (
+	"log/slog"
+)
+
+// LevelTrace must be added, because [slog] package does not have one by default. Generate it by subtracting 4 levels from [slog.Debug] following the example of [slog.LevelWarn] and [slog.LevelError] which are set to 4 and 8.
+const LevelTrace = slog.LevelDebug - 4
```

Comment:
Can this be made unexported? 🤔 This lives in the top-level watermill package, so it would be best to keep unnecessary symbols out of it.

Reply:
Oh, looking at the test, I assume it might be useful for configuring Logger to follow the trace level 👍